auth/services/oauth/oauth.go

```diff
@@ -273,6 +274,12 @@ func (s *service) CreateJwtGrant(request services.CreateJwtGrantRequest) (*servi
 		return nil, err
 	}
 
+	for _, verifiableCredential := range request.Credentials {
+		if err := credential.Validate(verifiableCredential); err != nil {
+			return nil, fmt.Errorf("invalid VerifiableCredential: %w", err)
+		}
+	}
+
 	endpointID, _, err := services.ResolveCompoundServiceURL(s.docResolver, *custodian, request.Service, services.OAuthEndpointType, nil)
```

Comment:
`NutsAuthorizationCredential` specific part is not implemented yet?

> The name and city MUST exactly (case sensitive) match with a valid Verifiable Credential issued to the actor. This must be a NutsOrganizationCredential.

(RFC003 5.2.1.5 )

Reply:
See: https://github.com/nuts-foundation/nuts-specification/pull/119

Reply:
Ah right I missed that.